src/components/actionButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { IonCard, IonIcon } from "@ionic/react";
+import { arrowForward } from "ionicons/icons";
+import React from "react";
+
+interface ActionButtonProps {
+  routerLink: string;
+  icon?: any;
+  text: string
+}
+
+export const ActionButton: React.FC<ActionButtonProps> = ({ routerLink, icon, text }) => {
+  return (
+    <IonCard button routerLink={routerLink} mode="ios" className="action-button shadow ion-padding ion-no-margin chapter">
```

Comment:
Why `.chapter`? I think `action-button` should be sufficient. The ion-padding is too much, I think drop that and add custom padding on the button itself. I'll put comments in the CSS file.

You don't need `mode="ios"`, that is now set globally.

Reply:
.chapter was included because it's already styled to noto serif and I didn't want to repeat the styling under .action-button. I can add it though if you prefer that